my_list.rb

```diff
@@ -0,0 +1,12 @@
+require './my_enumerable'
```

Comment:
Please use `require_relative` as using `require_relative` for files in your directory instead of `require` is a ruby convention. Please use `require` for external files, like gems, while `require_relative` should be used for referring to files within your directory.